tests/knownIssues.js

```diff
@@ -33,8 +35,10 @@ module.exports = [
 	// this implementation has numerous edit experience issues
 	'TodoMVC - angularjs_require, Mark all as completed, should allow me to mark all items as completed',
 	'TodoMVC - angularjs_require, Mark all as completed, should allow me to clear the completion state of all items',
+	//jscs:disable
```

Comment:
`// jscs:disable`, generally we do that space after a comment. Or do you want to handle JSCS comments as an exception?